js/admin-startPage.js

```diff
@@ -0,0 +1,54 @@
+$(function(){
+
+    axios.get("https://hakimlivs.herokuapp.com/customerOrder/orders")
+    .then(response => {
+        getOrderStatus(response.data)
+        //console.log(response.data)
+    })
+
+    function getOrderStatus (data) {
+        
+        let waiting = 0;
+        let picking = 0;
+        let ready = 0;
+        let sent = 0;
+        let canceled = 0;
+            data.forEach(element => {
+                console.log(element.orderStatus.type)
+                switch(element.orderStatus.type){
+                    case "Ohanterad": waiting += 1;  break;
+                    case "Plockning pågår": picking += 1;       break; 
```

Comment:
Heter "plockas" nu

Reply:
Nu har jag dragit in senaste main så den är uppdaterad till "plockas"